codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSSmokeTestGenerator.kt

```diff
@@ -0,0 +1,77 @@
+package software.amazon.smithy.aws.swift.codegen
```

Comment:
Extends generic smoke test generator in smithy-swift. Overrides methods to provide AWS specific logic.